static/app/stores/pageFiltersStore.spec.tsx

```diff
@@ -36,6 +36,12 @@ describe('PageFiltersStore', function () {
     });
   });
 
+  it('returns a stable reference with getState', () => {
+    PageFiltersStore.updateProjects([1], []);
+    const state = PageFiltersStore.getState();
+    expect(Object.is(state, PageFiltersStore.getState())).toBe(true);
```

Comment:
This assertion could be `.toBe()` which uses `Object.is` internally, and will give better messages if the assertion fails

https://jestjs.io/docs/expect#tobevalue

Reply:
yeah i'm trying to be explicit on what is being tested. fear people would see it fail and change it to toEqual